src/main/java/org/folio/search/service/FacetService.java

```diff
@@ -37,17 +39,26 @@ public FacetResult getFacets(CqlFacetRequest request) {
     searchSource.size(0).from(0).fetchSource(false);
 
     facetQueryBuilder.getFacetAggregations(request, searchSource.query()).forEach(searchSource::aggregation);
-    cleanUpFacetSearchSource(searchSource);
+    cleanUpFacetSearchSource(searchSource, List.of("items.effectiveLocationId"));
```

Comment:
Would be good to move to a constant.
It was mentioned to keep also tenantId filter.